### PR TITLE
[17.0][IMP] product_category_pricelist_button: moved legend from Pricelist Items to Price Rules

### PR DESCRIPTION
**Pull request by Anxo82**
Change string field, translations and new README.

### CODE REVIEW

**Comment by Anxo82** [comment]
@ChristianSantamaria @dalonsod Review please.
Thanks

File: product_category_pricelist_button/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Product category pricelist button",
     "summary": """
-        Adds a smart button to product categories linking to associated pricelists items.
+        Adds a smart button to product categories linking to associated price rules.
         """,
     "author": "Solvos",
     "license": "LGPL-3",
```
Comment (ChristianSantamaria) [inline product_category_pricelist_button/__manifest__.py:10]: Miss manifest version

**Review by ChristianSantamaria — CHANGES_REQUESTED** [review]
LGTM, only a improvement:

**Comment by Anxo82** [comment]
@ChristianSantamaria done!
Thanks

**Comment by dalonsod** [comment]
Tested :+1: 

I've renamed commit and PR title. @Anxo82 for those `[IMP]` or `[FIX]` PR/commits extra information within main title is needed, "[IMP] addon_name" is not enough.

Let's merge...

**Pull request merged**